File: frontend/src/api/useAutoApi/relationGraph.js
import request from '/@/utils/request';

/**
 * 项目使用接口
 * @method getProjectList 获取项目列表
 * @method getMenuTest 获取后端动态路由菜单(test)
 */
export function useRelationGraphApi() {
	return {
		getRelationGraph: (data) => {
			return request({
				url: '/relationGraph/getRelationGraph',
				method: 'POST',
				data,
			});
		},

	};
}

File: backend/autotest/apis/api/relation_graph.py
# -*- coding: utf-8 -*-
# @author: xiao bai
from fastapi import APIRouter

from autotest.schemas.api.relation_graph import RelationGraphService, RelationIn
from autotest.utils.response.http_response import partner_success

router = APIRouter()


@router.post('/getRelationGraph', description="关系图")
async def api_case_list(params: RelationIn):
    data = await RelationGraphService.get_relation(params)
    return partner_success(data)
